src/background/service/APIService.js

```diff
@@ -808,7 +808,7 @@ class APIService {
             let u8Hash = await oasis.hash.hash(oasis.misc.toCBOR(txWrapper.unverifiedTransaction))
             let hash = oasis.misc.toHex(u8Hash)
             let config =  getRuntimeConfig(params.runtimeId)
-            if (hash && config.accountType === RUNTIME_ACCOUNT_TYPE.EVM) {
```

Comment:
What was wrong with this approach?

Also does that PR needs to address another use cases like 
`let isEmerald = runtimeConfig.accountType === RUNTIME_ACCOUNT_TYPE.EVM`  where now we have multiple EVMc paratimes?

Reply:
Do you mean with `if (hash && config.accountType === RUNTIME_ACCOUNT_TYPE.EVM && config.runtimeName === "Emerald") {` approach? There's no difference

I think all other conditions are okay. (but `isEmerald` is a bad name - and there's already `isEvm` too :/ )